### PR TITLE
Clarify and reference spinbit optional/version

### LINKED ISSUE #179: Sect 3.7.2 : Is the spinning of bits invariant or for the current QUIC Spec?
Sect 3.7.2 : “is optional in QUIC”. 

I think it would be helpful to say more about the spin-bit definition for the current version of the specification, and a cross-reference is needed here. 
Similarly, “An endpoint not participating...” to me this description applies to the current specification, not an invariant of QUIC in general.

### PR DESCRIPTION
**Pull request by britram**
Fixes #179.

### CODE REVIEW

**Comment by britram** [comment]
fix #179

**Review by mirjak — APPROVED** [review]

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -719,7 +719,8 @@ handshake) at both sides.
 
 The spin bit provides an additional method to measure per-flow RTT from
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:720]:
```suggestion
The spin bit provides an additional version-specific method (see section 17.4 of {{?QUIC-TRANSPORT}}) to measure per-flow RTT from
```

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -719,7 +719,8 @@ handshake) at both sides.
...
 observation points on the network path throughout the duration of a connection.
-Endpoint participation in spin bit signaling is optional in QUIC. That is, while
+Endpoint participation in spin bit signaling is optional and version-specific
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:722]:
```suggestion
Endpoint participation in spin bit signaling is optional. That is, while
```

File: draft-ietf-quic-manageability.md [hunk 3]
```diff
@@ -719,7 +719,8 @@ handshake) at both sides.
...
+(see section 17.4 of {{?QUIC-TRANSPORT}}). That is, while
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:723]:
```suggestion
```

**Review by mirjak — CHANGES_REQUESTED** [review]

**Pull request merged**